vm/proxyapp/proxyappclient_mocks_test.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2022 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package proxyapp
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/google/syzkaller/vm/proxyapp/proxyrpc"
+	"github.com/stretchr/testify/mock"
+)
+
+type MockProxyApp struct {
+	mock   mock.Mock
+	config *MockProxyAppConfig
+	size   int
```

Comment:
Unused. Remove.

Reply:
Removed size and config. Cleanup was needed.